### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by tomjelinek**
* remove unnecessary quotes and backslashes
* readme readability improvements
* add missing spdx headers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -348,78 +348,78 @@ all:
...
+                value: 'true'
         operations:
-          - action: "start"
+          - action: start
             attrs:
-              - name: "timeout"
-                value: "30"
-          - action: "monitor"
+              - name: timeout
+                value: 30
```
Comment (richm) [inline README.md:380]: This changes the "type" from string to integer - where are these values used?  If the only place they are used is with `{{ value }}` then that should be ok - that will convert to string

Comment (tomjelinek) [reply README.md]: Yes, they are only used in `{{ value }}`. In general, they are all strings, though. So I fixed that and added an example and a test for setting timeout and intervals with time units.

**Comment by tomjelinek** [comment]
[citest bad]

**Comment by tomjelinek** [comment]
[citest bad]

**Review by richm — APPROVED** [review]
lgtm

**Comment by richm** [comment]
Please use the Squash and merge button to merge this

**Pull request merged**